target/linux/ipq807x/base-files/etc/board.d/02_network

```diff
@@ -6,6 +6,31 @@
 . /lib/functions/uci-defaults.sh
 . /lib/functions/system.sh
 
+mtd_chardev_get_mac_ascii() {
```

Comment:
Can't ```mtd_get_mac_ascii``` in system.sh be used?

Reply:
Since its being read from U-boot env then NVMEM U-boot driver with the custom OpenWrt ASCII cell can be used to completely drop this custom code

Reply:
`mtd_get_mac_ascii` tries to access the mtdblock device (/dev/mtdblock7), which triggers a kernel warning. This functtion is intended to explicitly use /dev/mtd7

@robimarko, I am not aware of this new driver

Reply:
https://github.com/openwrt/openwrt/blob/master/target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-nbg7815.dts#L217

And then you can see the users in the same DTS

Reply:
Is there a way to use it with "qcom,smem-part", without having to resort to "fixed-partitions" ?

Reply:
example: https://github.com/openwrt/openwrt/pull/11905/files#diff-905745526158b7213db1498962ee8e3f5a846519340fdbb4668379fec0aa77bdR191-R210